### ISSUE TITLE
This does the opposite work

### ISSUE THREAD

**Issue by vzenzo** [Sycnex/Windows10Debloater#472]
I've ran your Debloater and I will say, you can easily call it Bloater because once I ran it, restarted my PC, all the bloat was INSTALLED

I've already removed some of the things from my Windows 10
Downloaded your "debloater", followed instructions, ran it, restart computer and all of it was there... Cortana, Tips, Weather, News, Maps, Solitare, Snip, etc etc etc....

Thanks...for bloating my pc again

**Comment by tort-oise** [comment]
You sure you ran it correctly?

**Comment by Kuenlun** [comment]
I don't think it does the opposite but Cortana, Weather, News, Maps were not removed from my computer.
Any idea why?
